### PR TITLE
Update to Nexus 3.21.2-03

### PR DESCRIPTION
**Pull request by bdellegrazie**
Small PR to update to latest version of Nexus
Thanks for doing this - neat solution to this problem!

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -10,9 +10,9 @@ ADD src ./src
 
 RUN mvn verify -T1C
 
-FROM sonatype/nexus3:3.19.1
+FROM sonatype/nexus3:3.21.2
 
-ARG NEXUS_CASC_VERSION=3.19.1-01
+ARG NEXUS_VERSION=3.21.2-03
```
Comment (sergei-ivanov) [inline Dockerfile]: I think it should remain `NEXUS_CASC_VERSION` here. We can probably introduce another ARG for `NEXUS_VERSION` just before the FROM statement.

Comment (bdellegrazie) [reply Dockerfile]: Good spot, thank you - fixed!

**Review by sergei-ivanov — APPROVED** [review]

**Comment by hasusuf** [comment]
@sventschui Since this PR been around for a long time, should I consider this plugin inactive and unmaintainable?

**Comment by bdellegrazie** [comment]
@hasusuf we forked it here:
https://github.com/AdaptiveConsulting/nexus-casc-plugin
that is currently maintained and up-to-date

**Comment by hasusuf** [comment]
@bdellegrazie Thanks!

**Comment by sventschui** [comment]
Seems that this Pr drowned in my GitHub notifications. Sorry for that! In fact I‘m no longer using nexus myself so the best way would be to go with the fork @bdellegrazie mentioned. 

@bdellegrazie is it ok for you if I add your fork to the readme?

**Comment by bdellegrazie** [comment]
Please do

**Comment by sventschui** [comment]
Closing this PR as the repo is not maintained anymore

**Pull request closed without merging**